src/Components/ArtworkFilter/ArtworkFilters/ResultsFilter.tsx

```diff
@@ -4,15 +4,44 @@ import {
   type UseFilterSelectResultsProps,
   useFilterSelectResults,
 } from "./Utils/useFilterSelectResults"
+import type {
+  FilterSelectChangeState,
+  Item,
+} from "@artsy/palette/dist/elements/FilterSelect/Components/FilterSelectContext"
 
 interface ResultsFilterProps extends UseFilterSelectResultsProps {
   expanded?: boolean
   placeholder: string
+
+  /**
+   * If client-side keyword filtering should operate against something other
+   * than simply the `item.label`, provide a function that returns the text
+   * to filter against.
+   */
+  searchableText?: (item: Item) => string
+
+  /**
+   * If true, user will be able to select all currently displayed filtered items
+   */
+  enableSelectAll?: boolean
+  onSelectAll?: (state: FilterSelectChangeState) => void
```

Comment:
All these new capabilities from https://github.com/artsy/palette/pull/1417 are opt-in. 

**Artwork Location** is the first facet to use them.